notebooks/community/model_garden/model_garden_pipeline_templates_t5x.ipynb

```diff
@@ -0,0 +1,850 @@
+{
```

Comment:
<p>Should we also clean up the Artifact Registry repo created in the notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1816/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>